ocn_005_historical_cyclone_intensity/README.md

```diff
@@ -0,0 +1,15 @@
+## Historical Cyclone Intensity Dataset Pre-processing
+This file describes the data pre-processing that was done to the [Tropical cyclones wind speed buffers footprint 1970-2009](https://preview.grid.unep.ch/index.php?preview=data&events=cyclones&evcat=4&lang=eng) for [display on Resource Watch](https://resourcewatch.org/data/explore/).
+
+This dataset is provided by the source as a (zipped) GeoTIFF file. The following variables are shown on Resource Watch:
+- 1970-2009 Maximum Storm Intensity (cy_intensity): Highest estimated category on the Saffir-Simpson hurricane wind scale, which measures storm intensity, during 1970-2009.
+
+Because the data were already provided in a compatible format and projection, very little processing was necessary to display this data on Resource Watch.
```

Comment:
```suggestion
To process this data for display on Resource Watch, the data in this NetCDF variable was converted to a GeoTIFF file so that it could be uploaded to Google Earth Engine.
```

Reply:
The source provides a zip file that contains a GeoTIFF in the correct projection. No processing of the file itself was necessary.